client/actions/tests/utils.js

```diff
@@ -40,7 +40,9 @@ export class HeartbeatEmitter {
 
   emit(eventName, data) {
     if (this.eventData[eventName] !== null) {
-      throw new Error(`Cannot emit a Heartbeat event more than once: ${eventName}`);
+      throw new Error(
+        `Cannot emit a Heartbeat event more than once: ${eventName}`,
+      );
```

Comment:
bloody Prettier messing with the lines I didn't intend to. Want me to manually revert this?